src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/convert_color_nv12.cpp

```diff
@@ -4,55 +4,91 @@
 
 #include <vector>
 
-#include "single_layer_tests/convert_color_nv12.hpp"
+#include "single_op_tests/convert_color_nv12.hpp"
 #include "common_test_utils/test_constants.hpp"
 
-using namespace LayerTestsDefinitions;
-
 namespace {
+using ov::test::ConvertColorNV12LayerTest;
+
+const std::vector<ov::Shape> in_shapes = {
+    {1, 20, 10, 1}
+};
 
-const std::vector<ov::Shape> inShapes_nhwc = {
-    {1, 10, 10, 1}
+auto generate_input_static_shapes = [] (const std::vector<ov::Shape>& original_shapes, bool single_plane) {
+    std::vector<std::vector<ov::Shape>> result_shapes;
+    for (const auto& original_shape : original_shapes) {
+        std::vector<ov::Shape> one_result_shapes;
+        if (single_plane) {
+            auto shape = original_shape;
+            shape[1] = shape[1] * 3 / 2;
+            one_result_shapes.push_back(shape);
+        } else {
+            auto shape = original_shape;
+            one_result_shapes.push_back(shape);
+            auto uvShape = ov::Shape{shape[0], shape[1] / 2, shape[2] / 2, 2};
+            one_result_shapes.push_back(uvShape);
+        }
+        result_shapes.push_back(one_result_shapes);
+    }
+    return result_shapes;
 };
 
+auto in_shapes_single_plain_static     = generate_input_static_shapes(in_shapes, true);
+auto in_shapes_not_single_plain_static = generate_input_static_shapes(in_shapes, false);
+
 const std::vector<ov::element::Type> inTypes = {
         ov::element::u8, ov::element::f32
 };
 
-const auto testCase_values = ::testing::Combine(
-    ::testing::ValuesIn(inShapes_nhwc),
+const auto test_case_values_single_plain = ::testing::Combine(
+    ::testing::ValuesIn(ov::test::static_shapes_to_test_representation(in_shapes_single_plain_static)),
     ::testing::ValuesIn(inTypes),
     ::testing::Bool(),
+    ::testing::Values(true),
+    ::testing::Values(ov::test::utils::DEVICE_CPU)
+);
+
+const auto test_case_values_not_single_plain = ::testing::Combine(
+    ::testing::ValuesIn(ov::test::static_shapes_to_test_representation(in_shapes_not_single_plain_static)),
+    ::testing::ValuesIn(inTypes),
     ::testing::Bool(),
+    ::testing::Values(false),
     ::testing::Values(ov::test::utils::DEVICE_CPU)
 );
 
+INSTANTIATE_TEST_SUITE_P(smoke_TestsConvertColorNV12SinglePlain,
+                         ConvertColorNV12LayerTest,
+                         test_case_values_single_plain,
+                        ConvertColorNV12LayerTest::getTestCaseName);
 
-INSTANTIATE_TEST_SUITE_P(smoke_TestsConvertColorNV12, ConvertColorNV12LayerTest, testCase_values, ConvertColorNV12LayerTest::getTestCaseName);
+INSTANTIATE_TEST_SUITE_P(smoke_TestsConvertColorNV12NotSinglePlain,
```

Comment:
May it be better to name it `smoke_TestsConvertColorNV12_3Plain`? Or just `smoke_TestsConvertColorNV12`